source/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8
+
+WORKDIR /app
+
+COPY ["file_compare.py", "flask_file_compare.py", "requirements.txt", "/app/"]
+
+RUN pip3 install -r ./requirements.txt
+
+RUN mkdir -p /app/input
```

Comment:
Why we need to create these folder

Reply:
ok, I will check, can we get the expected result without creating the directory inside conatiner